packages/regenerator-transform/src/emit.js

```diff
@@ -34,9 +34,11 @@ function Emitter(contextId) {
   // that have been marked as branch/jump targets.
   this.marked = [true];
 
+  this.insertedLocs = new Set();
```

Comment:
not sure what version of node do u want to support here, should I just leave it or import a ponyfill?

Reply:
I'm comfortable with only supporting Node 4+.